apis/nucleus/src/index.js

```diff
@@ -439,12 +457,13 @@ function nuked(configuration = {}) {
             if (this._instance) {
               throw new Error(`Field or object ${fieldName || qId} already mounted`);
             }
+            const onPopoverClose = () => this.emit('closePopover');
             this._instance = ListBoxPortal({
               element,
               app,
               fieldIdentifier,
               qId,
-              options: getOptions(options),
+              options: getOptions({ ...options, onPopoverClose }),
```

Comment:
This will override `options.onPopoverClose` if user defined that. Is it intended?

Reply:
`options` variable directly goes into the `ListBoxInline` component and inside of that component, in it's props => seems like no variable extracted as `onPopoverClose` : https://github.com/qlik-oss/nebula.js/blob/5873e58cf225e4c843f1cb1f098ccdabe91c6826/apis/nucleus/src/components/listbox/ListBoxInline.jsx#L46

so I think it should be safe to have that property in there. (OR am i missing something?)

Reply:
Sorry, I was a bit unclear. I think it is fine to send it in. I only meant that you may (or may not) want to change the order to:
```
{ onPopoverClose, ...options }
```
so that the user (who sets the `options`) can specify: `options.onPopoverClose` to define a custom handler, if we want that now or in the future. A minor detail since that property it is not exposed yet to the user.